ports/cpp/README.md

```diff
@@ -0,0 +1 @@
+# ANTLRv4 C3 C++ Port
```

Comment:
Can you please add a bit more information, like required C++ and CMake versions and some instructions to build and run the tests?

Reply:
Done.